app/assets/stylesheets/legacy/filters.scss

```diff
@@ -60,7 +60,7 @@ a.filter-option:visited,
 
 .dropdown-text-input {
   width: 600px;
-  padding: $default-vertical-margin/2 $grid-gutter-width/2;
+  padding: calc($default-vertical-margin / 2) calc($grid-gutter-width / 2);
```

Comment:
I'd have gone with `.div` here too. Does `div` not simplifying have much impact?

Reply:
As discussed, not sure about this, but going to defer to the approach of other GOV.UK apps, particularly from the frontend team